### PR TITLE
version: 0.0.27

### PR DESCRIPTION
**Pull request by AllRWeak**

### CODE REVIEW

**Review by at3rva — CHANGES_REQUESTED** [review]

File: nova/core/roles/customization_pre_role/tasks/main.yml [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
     path: "{{ inventory_dir }}/roles/{{ pre_role }}"
-  register: role_existence
+  register: pre_role_existence
   become: false
   delegate_to: localhost
 
-- name: Including local role {{ pre_role }}...
+- name: Including local {{ pre_role }} role...
   ansible.builtin.include_role:
     name: "{{ inventory_dir }}/roles/{{ pre_role | default(omit) }}"
```
[the remark this answers is not in the document]
Comment (AllRWeak) [reply nova/core/roles/customization_pre_role/tasks/main.yml]: Yes, pushed updates

File: nova/core/roles/customization_pre_role/tasks/main.yml [hunk 2]
```diff
@@ -1,20 +1,20 @@
 ---
-- name: Checking if the local role exists...
+- name: Checking if {{ pre_role }} exists in {{ customization_pre_role_local_root_folder }}/{{ pre_role }}...
   ansible.builtin.stat:
     path: "{{ inventory_dir }}/roles/{{ pre_role }}"
```
[the remark this answers is not in the document]
Comment (AllRWeak) [reply nova/core/roles/customization_pre_role/tasks/main.yml]: Yes, pushed updates

File: nova/core/roles/customization_single_role/tasks/main.yml [hunk 3]
```diff
@@ -1,35 +1,28 @@
...
+- name: Creating post scripts folder and including local role...
   when: local_role_existence.stat.exists
-
-- name: Including shared role {{ single_role }}...
+  block:
+    - name: Making sure that {{ post_scripts_folder_on_target }} folder exists...
+      ansible.windows.win_file:
+        path: "{{ post_scripts_folder_on_target }}"
+        state: directory
+      when: ansible_system == "Win32NT"
```
[the remark this answers is not in the document]
Comment (AllRWeak) [reply nova/core/roles/customization_single_role/tasks/main.yml]: Yes, pushed updates

File: nova/core/roles/customization_single_role/tasks/main.yml [hunk 4]
```diff
@@ -1,35 +1,26 @@
...
 
-- name: Checking if the shared role exists...
-  ansible.builtin.stat:
-    path: "{{ single_role }}"
-  register: shared_role_existence
-  become: false
-  delegate_to: localhost
-
-- name: Making sure that {{ post_scripts_folder_on_target }} folder exists for Windows...
+- name: Making sure that {{ post_scripts_folder_on_target }} folder exists...
```
Comment (at3rva) [inline nova/core/roles/customization_single_role/tasks/main.yml]: I think for this task we still need to check if the shared_role exist and then add to this task another when: `local_role_existence.stat.exists or shared_role_existence.stat.exists`

Maybe same should be config in `nova/core/roles/customization_pre_role/tasks/main.yml`

**Review by at3rva — APPROVED** [review]

**Pull request merged**